src/main/java/eggventory/parsers/Parser.java

```diff
@@ -46,6 +47,20 @@ public static boolean isReserved(String input) {
         return reservedNames.contains(input);
```

Comment:
nice nice
i assume thsi will be implemented later

Reply:
This has already been implemented for `add stocktype all` and `edit stocktype all`. Can be used for other commands too but haven't looked too deep into it yet.